src/phoenix/utilities/__init__.py

```diff
@@ -1,26 +0,0 @@
-from datetime import datetime
-from typing import List, Optional
-
-import pandas as pd
-
-from phoenix.core.project import Project
-from phoenix.trace.dsl import SpanQuery
-
-
-def query_spans(
```

Comment:
This get refactored?

Reply:
yes. this is moved from trace/otel.py and made into public functions

Reply:
this is no longer needed